app/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,7 @@
     <string name="hover_time_text">Hover Time</string>
     <string name="cursor_sensitivity_text">Cursor Sensitivity</string>
     <string name="hover_time_amount_text">%s seconds</string>
+    <string name="hover_time_one_text">1 second</string>
```

Comment:
You could potentially provide this as a [plurals](https://developer.android.com/guide/topics/resources/string-resource#Plurals) resource

Reply:
Yeah this got a little weird with plurals since it's a float value. So we followed the advice in this thread: https://stackoverflow.com/a/55006692